modules/http/server/src/test/scala/GreeterRestServices.scala

```diff
@@ -16,18 +16,20 @@
 
 package freestyle.rpc.http
 
+import cats.MonadError
 import cats.effect._
 import cats.syntax.flatMap._
 import cats.syntax.functor._
-import fs2.Stream
-import io.circe.Decoder
+import freestyle.rpc.http.Utils._
 import io.circe.generic.auto._
```

Comment:
Same questions here, about encoders/decoders

Reply:
Here I'm not sure where we'd be receiving the encoders from if we didn't define them here. Note that as with `GreeterRestClient` above, this class and its decoders would be auto-generated from the service annotations.